types/src/system.rs

```diff
@@ -7,7 +7,7 @@ pub mod mint;
 pub mod standard_payment;
 mod system_contract_type;
 
-pub use call_stack_element::CallStackElement;
+pub use call_stack_element::{CallStackElement, CallStackElementTag};
```

Comment:
Is this an accidental import? If not, what is its purpose?

Reply:
It was (accidentally?) removed despite being part of the actual public API in 1.4.6 and in `dev`.  Not sure why this isn't picked up in lints though.